lib/index.js

```diff
@@ -13,10 +13,10 @@ const TaskState = require('./models/TaskState');
 const bulletStyleSetup = require('./bulletStyleSetup');
 const quit = require('./quit');
 
-const tasksFile    = path.join(os.homedir(), 'lol-tasks.json');
-const settingsFile = path.join(os.homedir(), 'lol-settings.json');
-
 const term = terminal;
+const yaml = require('js-yaml');
```

Comment:
Imports should be at the top of the module. It's even better to group core Node.js libraries (like `fs`, `os`, `path`), 3rd party libraries and application module imports.

The line which imports `js-yaml` should be on line 1. Sadly GitHub review tool doesn't allow me to suggest that change. After reviewing other suggestions, this change would have to be committed separately.